website/data/docs-nav-data.json

```diff
@@ -1437,17 +1437,17 @@
         "title": "Overview",
         "path": "devices"
       },
-      {
-        "title": "Nvidia",
-        "path": "devices/nvidia"
-      },
       {
         "title": "Community",
         "routes": [
           {
             "title": "Overview",
             "path": "devices/external"
           },
+          {
+            "title": "Nvidia",
+            "path": "devices/external/nvidia"
+          },
```

Comment:
Is the `Community` title accurate for Nvidia?

Perhaps since we have no internal devices at this point we should remove the intermediary `Community` level from the TOC but keep the `devices/external/...` paths. That way if we ever bake in another device we don't have to break URLs to recreate the nested structure.

The 2 overviews ([top](https://www.nomadproject.io/docs/devices) and [community](https://www.nomadproject.io/docs/devices/external)) would need to be combined, but they're both quite short.

Reply:
Sure, that works.

Reply:
Hrm, unfortunately it looks like the web framework won't let us have mismatched depths for navigation items. From [here](https://vercel.com/hashicorp/nomad/HpPyb65t9WAfTAs724GBd4oR5ehx):

> Error: Invalid path depth. At depth 1, found path "devices/external/nvidia". Please move this path to the correct depth of 2.

Reply:
Oh well. Maybe change `Community` to `External` and call it a day?

Reply:
Works for me.

Reply:
Done.